todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+from pymongo import ASCENDING
+import pymongo
+from todo_app.data.itemStatus import ItemStatus
+from todo_app.data.todo_item import TodoItem
+from todo_app.mongo_config import MongoDBConfig
+from pymongo.errors import PyMongoError
+
+class MongoDbManager:
+
+    def __init__(self, config):
+        self.config = config
+        self.client = pymongo.MongoClient(config.PRIMARY_CONNECTION_STRING)
+        self.db = self.client[config.MONGO_DB_NAME]
+        self.collection = self.db[config.COLLECTION_NAME]
+        
+    def create_todo_item(self, name, description):
+        """
+        Create a new MongoDB document.
+
+        Args:
+          name: The name of the item.
+          description: The description of the item.
+        """
+        document = {
+            'name': name,
+            'description': description,
+            'status': ItemStatus.TODO.value
+        }
+        
+        result = self.collection.insert_one(document)
+        if result.acknowledged:
+            print(f"Document created with ID: {result.inserted_id}")
+        else:
+            print("Failed to create document")
+            
+    def get_todo_items(self) -> List[TodoItem]:
+        """
+        Get MongoDB documents and sort by status in ascending order.
+
+        Returns:
+          list: The list of saved items.
+        """
+        try:
+          # Check if an index exists on 'status' field 
+          if not self.collection.index_information().get("status_1"):
```

Comment:
Nothing wrong with this, and definitely a good thing in terms of working out how to do it, but for a real project I'd challenge whether this index adds anything for us here - it probably helps the sorting be sped up, but given that the ViewModel filters out the different statuses anyway I suspect we're not gaining much here! (i.e. we could remove this index and the sort stage, and I can't see that it would matter)